drivers/dma/Kconfig.mcux_edma

```diff
@@ -0,0 +1,29 @@
+# DMA configuration options
+
+# Copyright (c) 2020, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config DMA_MCUX_EDMA
+	bool "Enable MCUX DMA driver"
+	depends on HAS_MCUX_EDMA
+	select NOCACHE_MEMORY if HAS_MCUX_CACHE
+	help
+	  DMA driver for MCUX series SoCs.
+
+if DMA_MCUX_EDMA
+
+config DMA_TCD_QUEUE_SIZE
+	int "number of TCD in a queue for SG mode"
+	default 2
+	help
+	  number of TCD in a queue for SG mode
+
+config DMA_TEST_SLOT_START
```

Comment:
I wonder if this one shouldn't be moved out of this file to the generic Kconfig.dma.

Reply:
this is only required for kinetis, which test channel starts from 58, other platfrom like RT or ST SOC starts from 0

Reply:
@FRASTM Didn't we discuss something similar ?

Reply:
> this is only required for kinetis, which test channel starts from 58, other platfrom like ST SOC starts from 0, which does not need such special configure.

In such case, it would have to be DMA_MCUX_TEST_SLOT_START to make it clear.

Reply:
In some stm32 socs, it counts from 1, some from 0.  
In the test application chan_blen_transfer, I had a loop on the chan_id to select one or a more channel to test on. Something like :
```
#define FIRST_DMA_CHANNEL	CONFIG_FIRST_DMA_CHANNEL
#define LAST_DMA_CHANNEL	CONFIG_LAST_DMA_CHANNEL

	for (chan_id = FIRST_DMA_CHANNEL;
	     chan_id <= LAST_DMA_CHANNEL;
	     chan_id++) {
		if (dma_config(dma, chan_id, &dma_cfg)) {
			printk("ERROR: transfer config\n");
			return;
		}
}
```

Reply:
I will change to DMA_MCUX_TEST_SLOT_START